lib/apimocker.js

```diff
@@ -82,17 +78,17 @@ apiMocker.createServer = (options = {}) => {
   apiMocker.middlewares.push(apiMocker.router);
 
   if (options.proxyURL) {
-    logger(`Proxying to ${options.proxyURL}`);
+    logger.info(`Proxying to ${options.proxyURL}`);
     const proxyOptions = {
-      forwardPath(req) {
-        logger(`Forwarding request: ${req.originalUrl}`);
+      proxyReqPathResolver(req) {
```

Comment:
Unrelated to logging, this addresses a warning from `express-http-proxy`:
"forwardPath and forwardPathAsync are DEPRECATED and should be replaced with proxyReqPathResolver"
https://github.com/villadora/express-http-proxy#readme

Reply:
Thanks for fixing that too!  Merging now...